tests/task_router/test_workflow_config.py

```diff
@@ -78,6 +78,57 @@ def test_from_json2(self):
             self.assertEqual(2, len(config.task_routing.filters))
             self.assertEqual(4, len(config.task_routing.default_filter))
 
+        def test_from_json_filter_friendly_name(self):
+
+            data = {
+                       'task_routing':
+                       {
+                           'filters': [
+                               {
+                                   'expression': 'type == "sales"',
+                                   'filter_friendly_name': 'Sales',
+                                   'targets': [
+                                       {
+                                           'queue': 'WQec62de0e1148b8477f2e24579779c8b1',
+                                           'expression': 'task.language IN worker.languages'
+                                       }
+                                   ]
+                               },
+                               {
+                                   'expression': 'type == "marketing"',
+                                   'filter_friendly_name': 'Marketing',
+                                   'targets': [
+                                       {
+                                           'queue': 'WQ2acd4c1a41ffadce5d1bac9e1ce2fa9f',
+                                           'expression': 'task.language IN worker.languages'
+                                       }
+                                   ]
+                               },
+                               {
+                                   'expression': 'type == "support"',
+                                   'filter_friendly_name': 'Support',
+                                   'targets': [
+                                       {
+                                           'queue': 'WQe5eb317eb23500ade45087ea6522896c',
+                                           'expression': 'task.language IN worker.languages'
+                                       }
+                                   ]
+                               }
+                           ],
+                           'default_filter':
+                           {
+                               'queue': 'WQ05f810d2d130344fd56e3c91ece2e594'
+                           }
+                       }
+                   }
+
+            config = WorkflowConfig.json2obj(json.dumps(data))
+            self.assertEqual(3, len(config.task_routing.filters))
+            self.assertEqual(1, len(config.task_routing.default_filter))
+            self.assertEqual("Sales", config.task_routing.workflow_rules[0].friendly_name)
+            self.assertEqual("Marketing", config.task_routing.workflow_rules[1].friendly_name)
+            self.assertEqual("Support", config.task_routing.workflow_rules[2].friendly_name)
```

Comment:
we should also test that the same workflow is marshaled back as friendly_name instead of filter_friendly_name